orange/src/main/java/com/hartwig/hmftools/orange/report/chapters/GermlineFindingsChapter.java

```diff
@@ -179,16 +180,16 @@ private void addGermlineCNAberrations(@NotNull Document document) {
                 germlineAberrationJoiner.add(germlineAberration.toString());
             }
             Table table = new Table(UnitValue.createPercentArray(new float[] { 1 })).setWidth(contentWidth());
-            table.addCell(Cells.createContent(germlineAberrationJoiner.toString()));
-            document.add(Tables.createWrapping(table, "Germline CN aberrations (" + count + ")"));
+            table.addCell(reportResources.cells().createContent(germlineAberrationJoiner.toString()));
+            document.add(reportResources.tables().createWrapping(table, "Germline CN aberrations (" + count + ")"));
         }
     }
 
-    private void addPharmacogenetics(@NotNull Document document) {
+    private void addPharmacogenetics(@NotNull Document document, @NotNull ReportResources reportResources) {
         Set<PeachGenotype> peach = report.peach();
         if (peach != null) {
             String titlePharmacogenetics = "Pharmacogenetics (" + peach.size() + ")";
-            document.add(PharmacogeneticsTable.build(titlePharmacogenetics, contentWidth(), peach));
+            document.add(PharmacogeneticsTable.build(titlePharmacogenetics, contentWidth(), peach, reportResources));
```

Comment:
@kduyvesteyn, I'll reply here for context and to facilitate discussion since github won't let me reply to your general comment.
> * Have no super strong opinion but I'd think reportResources should be a property of some classes rather than an input to the main function. Eg, the classes in "tables" package are not fundamentally different from Cells and Tables. In theory a pdf could generate the same table multiple times (with different inputs). The reportResources would be a property of the class generating the tables (exactly similar to Cells and Tables)

I started out by providing the resources to the `tables` classes in the constructor, but each of these classes has a private, no-argument constructor and all interactions are with the static `build` function. Taking your suggestion would mean changing all calls to build tables as follows:
```suggestion
            document.add(new PharmacogeneticsTable(reportResources).build(titlePharmacogenetics, contentWidth(), peach));
```
I find that clunkier to work with, and I don't think making the `ReportResources` a class property would improve much in the rare case where we call the same table class multiple times since we're just passing a reference either way.

On the other hand, adding a `ReportResources` property to the `ReportChapter` classes would eliminate the need to pass that parameter around through so many method calls, so I'm happy to refactor on that side.

Reply:
Changing the table classes would indeed not have that much impact. I also considered proposing changing the Chapter interface but then I think a Document is always tied to one ReportResources, so I think at least the interface now is consistent. If we make `reportResources` a property then maybe Document should be one as well, since we are not supposed to generate 2 different documents with one `reportResources`. 

In any case I'd also be fine with current approach. If you feel that's ok I suggest to leave as you initially made it.